src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php

```diff
@@ -368,7 +367,6 @@ private function renderEditProductForm(FormInterface $productForm, int $productI
             'productForm' => $productForm->createView(),
             'statsLink' => $this->getAdminLink('AdminStats', ['module' => 'statsproduct', 'id_product' => $productId]),
```

Comment:
@jolelievre Can you check this line? I am on develop and the button is displayed even if I uninstalled the related stat modules. Maybe we could check if the module is active. ;-)

Reply:
I added a check, the link is only added when module `statsproduct` is installed and enabled